estate/models/estate_property.py

```diff
@@ -0,0 +1,140 @@
+from odoo import models, fields, api, exceptions
+from datetime import timedelta
```

Comment:
```suggestion
from datetime import timedelta

from odoo import api, fields, models,
from odoo.exceptions import UserError
```
Generally we first make external imports and then we import from odoo in alphabetical order.
Need to adapt in all files